packages/ui/components/ui/tx/view/value.tsx

```diff
@@ -41,7 +41,7 @@ export const ValueViewComponent = ({
       <div className='flex items-center gap-1 text-base font-bold'>
         {showIcon && <AssetIcon metadata={metadata} />}
         {showValue && <span>{formattedAmount}</span>}
-        {showDenom && <span>{symbol}</span>}
+        {showDenom && <span className='truncate'>{symbol}</span>}
```

Comment:
Eventually, we'll probably want to re-label delegation tokens to be briefer, but this is good enough for now.

## Before
![image](https://github.com/penumbra-zone/web/assets/1121544/c38d5291-281c-462d-9694-03e340daaf96)


## After
![image](https://github.com/penumbra-zone/web/assets/1121544/0377ba90-b844-4188-beca-288f6e426e4b)

Reply:
Should we make this a bit bigger? Usually asset icons are the size of the text

<img width="74" alt="Screenshot 2024-02-27 at 9 09 46 AM" src="https://github.com/penumbra-zone/web/assets/16624263/80d789c1-bd7c-4d28-bbea-184aa53e9737">

Reply:
Whoops, will fix

Reply:
<img width="226" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/33d4470d-9925-49c3-9a5e-809e74cd562c">
[Fixed](https://github.com/penumbra-zone/web/pull/604/commits/d1ecc70552d251a71eb4af362d9848641d0c4d48)